agasc/tests/test_agasc_1.py

```diff
@@ -47,3 +48,60 @@ def test_multi_agasc():
     assert len(cone2) == len(cone2_full)
 
 
+def test_update_color1_func():
+    """
+    Test code to update the COLOR1 column.
+    """
+    color1 = [1.0, 1.0, 1.5, 1.5, 1.5, 1.5]
+    color2 = np.array([1.0, 1.0, 1.5, 1.5, 1.75, 2.0]) / 0.850
```

Comment:
My only nit here would be that color2 is not color2 / 0.850, and condensing this in to one line was a little problem for me with readability.  Not worth a change.